packages/matchbox/src/components/Spinner/styles.js

```diff
@@ -68,7 +94,12 @@ export const circle = props => {
 
     stroke-dasharray: ${strokeDashArray};
```

Comment:
We can probably remove this since setting the `stroke-dasharray` with `getDefaultStrokes`